### ISSUE TITLE
Installation Issue: Installing Python libraries

### ISSUE THREAD

**Issue by stephen-hills** [idaholab/raven#1246]
--------
Issue Description
While following the instructions for installationWindowsNew, I had a problem during the third step.  When on the step using the code: bash.exe scripts\establish_conda_env.sh --install --conda-defs <yourCondaDefsLocation> (where <yourCondaDefsLocation> is the replaced by my location) it is unable to find the conda environment: raven_libraries.  It says a couple of times that permission is denied.
--------
##### What did you expect to see happen?
I expected raven_libraries to be found and installed.

##### What did you see instead?
Unable to find the conda environment: raven_libraries.  It says a couple of times that permission is denied. See the picture below for the output.
![image](https://user-images.githubusercontent.com/52016458/83777789-0c736780-a647-11ea-9ea0-fe1d9ea37887.png)

##### Do you have a suggested fix for the development team?
No

##### Please attach the input file(s) that generate this error.  The simpler the input, the faster we can find the issue.

----------------
For Change Control Board: Issue Review
----------------
This review should occur before any development is performed as a response to this issue.
- [ ] 1. Is it tagged with a type: defect or task?
- [ ] 2. Is it tagged with a priority: critical, normal or minor?
- [ ] 3. If it will impact requirements or requirements tests, is it tagged with requirements?
- [ ] 4. If it is a defect, can it cause wrong results for users? If so an email needs to be sent to the users.
- [ ] 5. Is a rationale provided? (Such as explaining why the improvement is needed or why current code is wrong.)

-------
For Change Control Board: Issue Closure
-------
This review should occur when the issue is imminently going to be closed.
- [ ] 1. If the issue is a defect, is the defect fixed?
- [ ] 2. If the issue is a defect, is the defect tested for in the regression test system?  (If not explain why not.)
- [ ] 3. If the issue can impact users, has an email to the users group been written (the email should specify if the defect impacts stable or master)?
- [ ] 4. If the issue is a defect, does it impact the latest release branch? If yes, is there any issue tagged with release (create if needed)?
- [ ] 5. If the issue is being closed without a pull request, has an explanation of why it is being closed been provided?

**Comment by wangcj05** [comment]
@joshua-cogliati-inl Any suggestion on this issue? I think we do not see this issue recently, and I guess it is not raven related issue, but an issue with respect to the users right on their system.

**Comment by dgarrett622** [comment]
@wangcj05 I tried running this in various shell environments. Here is what I found:

The user that posted this issue was using the Windows Command Prompt. When I try running a very similar command I get the following error:
<img width="550" alt="cmd" src="https://user-images.githubusercontent.com/19575707/178305368-3cf4411b-a5fd-41a0-a460-1955a090d0a4.PNG">

When I try running a similar command in Windows PowerShell, I get the same error:
<img width="550" alt="powershell" src="https://user-images.githubusercontent.com/19575707/178305420-2a6a134a-5ff2-4fc8-9f1d-d3937a22df3f.PNG">

When running in GitBash, the script successfully runs.

Part of the problem is that bash scripts do not work very well on Windows in general. I have not been able to get other bash scripts to work using Command Prompt. I am not as familiar with PowerShell, but in theory, it should be able to handle Unix style bash scripts. I have had success using GitBash to run the bash scripts in RAVEN. Maybe we should change the documentation/wiki to tell the users to use GitBash when running RAVEN on Windows?
